httpreq/httpreq_test.go

```diff
@@ -6,10 +6,14 @@ import (
 	"fmt"
 	"net/http"
 	"net/http/httptest"
+	"os"
 	"testing"
+	"time"
 
 	"github.com/pangpanglabs/goutils/behaviorlog"
 
+	httpreq2 "goutils/httpreq"
```

Comment:
Do not use `goutils/httpreq` package.

Reply:
now,I use this package :"github.com/pangpanglabs/goutils/httpreq"